### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50942]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Authenticated 22-1990e Transfer of Entitlement (TOE) Form Updates

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
In the "Need Help" section, the TTY: 711 phone number should have "TTY" inside the link text and included in the aria-label.

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

### Platform Recommendation
The `va-telephone` component supports a `tty` property that properly formats TTY links.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by btruongafs** [comment]
@briandeconinck this issue has been fixed
